### DISCUSSION TITLE
json import fails

### DISCUSSION THREAD

**Discussion by vinothvk** [vitest-dev/vitest#1337] [Q&A] [answered]
Hello

Does vitest support import of json out of the box ?
My source code that imports a JSON file throws below error when i run vitest 

TypeError: Unknown file extension ".json"

Thanks

**Comment by Rvtea** [comment, marked as answer]
I workaround this via pass the ` --experimental-json-modules` to the command via something like `"test": "NODE_OPTIONS='--experimental-json-modules' vitest run"`

**Reply by sheremet-va** [reply]
Importing json should work in you source code, but will fail without the `experimental-json-modules` option, like mentioned above. You can also use `deps.inline` option in Vitest config to allow Vitest process dependency files.

**Reply by Rvtea** [reply]
thx for the suggestion, I do try `deps.inline` however it does not work for my case, so I use the workaround as current solution.
Some more context for my use case: struggled with `dayjs` and it output with one `locale.json`, vitest compile the files and turns out with the json import issue. I tried add `deps.inline` with `['dayjs']`, but it do not work. Do I use it wrong?

**Reply by sheremet-va** [reply]
Try `deps.inline: [/dayjs/]`

**Reply by vinothvk** [reply]
Thanks @Rvtea @sheremet-va both solutions work for me

**Reply by Rvtea** [reply]
@sheremet-va 
sorry I forgot to mention that I also tried `deps.inline: [/dayjs/]` and also not works. My config is like this:
```
test: {
  deps: {
    inline: [/dayjs/]
  }
}
```
